target_chains/cosmwasm/tools/src/instantiate-pyth.ts

```diff
@@ -0,0 +1,164 @@
+    desc: "Execute this script for mainnet networks. THIS WILL BE ADDED IN FUTURE",
+  })
+  .help()
+  .alias("help", "h")
+  .wrap(yargs.terminalWidth())
+  .parseSync();
+
+async function run() {
+  const STORAGE_DIR = "./testnet/instantiate-pyth";
+
+  // get the wasm code from github
+  let contractBytesDict = await getContractBytesDict(
+    Object.values(ExtendedChainsConfigTestnet).map(
+      ({ pythArtifactZipName }) => pythArtifactZipName
+    ),
+    argv.contractVersion
+  );
+
+  let chainIds = argv.chainId === undefined ? ChainIdsTestnet : [argv.chainId];
+  for (let chainId of chainIds) {
+    let pipelineStoreFilePath = `${STORAGE_DIR}/${chainId}-${argv.contractVersion}.json`;
+    const pipeline = new Pipeline(chainId, pipelineStoreFilePath);
+
+    const chainExecutor = createExecutorForChain(
+      ExtendedChainsConfigTestnet[chainId],
+      argv.mnemonic
+    );
+
+    // add stages
+    // 1 deploy artifact
+    pipeline.addStage({
+      id: "deploy-pyth-code",
+      executor: async () => {
+        return chainExecutor.storeCode({
+          contractBytes:
+            contractBytesDict[
+              ExtendedChainsConfigTestnet[chainId].pythArtifactZipName
+            ],
+        });
+      },
+    });
+
+    // 2 instantiate contract
+    pipeline.addStage({
+      id: "instantiate-contract",
+      executor: (getResultOfPastStage) => {
+        const storeCodeRes: StoreCodeResponse =
+          getResultOfPastStage("deploy-pyth-code");
+
+        return chainExecutor.instantiateContract({
+          codeId: storeCodeRes.codeId,
+          instMsg: getPythConfig({
+            feeDenom: ExtendedChainsConfigTestnet[chainId].feeDenom,
+            wormholeChainId:
+              ExtendedChainsConfigTestnet[chainId].wormholeChainId,
+            wormholeContract: getWormholeContractAddress(
+              chainId,
+              WORMHOLE_CONTRACT_VERSION,
+              false
+            ),
+          }),
+          label: "wormhole",
+        });
+      },
+    });
+
+    // 3 set its own admin
+    pipeline.addStage({
+      id: "set-own-admin",
+      executor: (getResultOfPastStage) => {
+        const instantiateContractRes: InstantiateContractResponse =
+          getResultOfPastStage("instantiate-contract");
+
+        return chainExecutor.updateContractAdmin({
+          newAdminAddr: instantiateContractRes.contractAddr,
+          contractAddr: instantiateContractRes.contractAddr,
+        });
+      },
+    });
+
+    await pipeline.run();
+  }
+}
+
+function getPythConfig({
+  feeDenom,
+  wormholeContract,
+  wormholeChainId,
+}: {
+  feeDenom: string;
+  wormholeContract: string;
+  wormholeChainId: number;
+}) {
+  return {
+    wormhole_contract: wormholeContract,
+    governance_source_index: 0,
+    governance_sequence_number: 0,
+    chain_id: wormholeChainId,
+    valid_time_period_secs: 60,
+    fee: {
+      amount: "1",
+      denom: feeDenom,
+    },
+    data_sources: [
+      {
```

Comment:
maybe comment here that all chains (including testnets) are using the mainnet data sources